src/lib/lore.sh

```diff
@@ -36,6 +45,13 @@ declare -gr SEPARATOR_CHAR='Æ'
 # it easy to undertand.
 declare -ag list_of_mailinglist_patches
 
+function create_lore_data_dir()
+{
+  [[ -d "${LORE_DATA_DIR}" ]] && return
+
+  mkdir -p "${LORE_DATA_DIR}"
+}
```

Comment:
I was wondering about this function... what if we cannot create LORE_DATA_DIR? Maybe we should add some extra checks for `mkdir` and handle the rare case where we cannot create this folder.